Misc/NEWS.d/next/Documentation/2024-11-26-00-09-16.gh-issue-127276.4ST1Wf.rst

```diff
@@ -0,0 +1 @@
+Add embedded styles to documentation pages created by a 'python -m pydoc -w' call
```

Comment:
I'm on mobile but this new feature warrants a What's New entry as well.

By the way, I'd suggest using back quotes to format the command line rather than single quotes as well as adding a period at the end of the sentence:

```rst
Allow to specify embedded styles for documentation pages created by :option:`pydoc -w`.
```

Please check that pydoc is documented as a program (with the `.. program::` directive) and that -w is also a documented option (with the `.. option::` directive). If not, you can just use 

```rst
... created by the ``-w`` option of the :mod:`pydoc` program.
```